dali/test/python/test_external_source_parallel_large_sample.py

```diff
@@ -49,9 +50,9 @@ def create_pipeline():
             a = np.array(out[idx_in_batch])
             assert a.shape == (512,), "Expected shape (512,) but got {}".format(a.shape)
             for val in a.flat:
-                assert val == expected_val, \
-                    "Unexpected value in batch: got {}, expected {}, for batch {}, sample {}".format(
-                        val, expected_val, batch_idx, idx_in_batch)
+                assert val == expected_val, (
+                    f"Unexpected value in batch: got {val}, expected {expected_val}, "
+                    f"for batch {batch_idx}, sample {idx_in_batch}")
```

Comment:
Is this parenthesis required for anything?
```suggestion
                assert val == expected_val, \
                    f"Unexpected value in batch: got {val}, expected {expected_val}, " \
                    f"for batch {batch_idx}, sample {idx_in_batch}"
```

Reply:
It's a matter of... preference. I found it surprising.